bin/bcs_weekly-report.sh

```diff
@@ -0,0 +1,226 @@
+        elif [ "$freq_type" -eq 1 ]; then
+            echo -e "\\pset footer off
+                     SELECT \
+array_to_string(array_reverse(string_to_array(r.path, '|')), '|') as path, \
+r.urban_quan, r.rural_quan, r.vers\
+                         FROM openquakeplatform_building_class_classificationhead as h,\
+                              openquakeplatform_building_class_classificationrow as r \
+                         WHERE h.id = r.head_id AND h.id = $header_id AND h.owner_id = $owner_id;" | \
+                tee -a $LOG_FILE | psql -q --html '--table-attr=class="bcs_items"' oqplatform
+        else
+            echo "ERROR: unknown frequency type"
+            exit 1
+        fi
+        echo "<p class='divfoot'></p>"
+        echo "</div>"
+    done
+    echo "<hr/>"
+done
+
+cat <<EOF
+</body>
+</html>
+EOF
+) > bcs_weekly-report_${DT}.html
+
+echo "\\copy (\
+SELECT u.username, u.first_name, u.last_name, u.email, c.name_0 as country, \
+\
+CASE h.occupancy WHEN 1 THEN 'residential'\
+                 WHEN 2 THEN 'commercial'\
+                 WHEN 4 THEN 'industrial'\
+                 WHEN 8 THEN 'educational'\
+                 WHEN 16 THEN 'healthcare'\
+                 WHEN 32 THEN 'governmental'\
+                 ELSE 'UNKNOWN !!' END as occupancy, \
+                 h.notes, h.vers, \
+\
+array_to_string(array_reverse(string_to_array(r.path, '|')), '|') as path, \
+CASE r.urban WHEN 0 THEN 'inexistent'\
+             WHEN 1 THEN 'rare'\
+             WHEN 2 THEN 'moderately freq.'\
+             WHEN 3 THEN 'frequent'\
+             WHEN 4 THEN 'very frequent'\
+             ELSE 'UNKNOWN !!' END as urban, \
+CASE r.rural WHEN 0 THEN 'inexistent'\
+             WHEN 1 THEN 'rare'\
+             WHEN 2 THEN 'moderately freq.'\
+             WHEN 3 THEN 'frequent'\
+             WHEN 4 THEN 'very frequent'\
+             ELSE 'UNKNOWN !!' END as rural, \
+r.urban_quan, r.rural_quan, r.vers \
+\
+FROM auth_user AS u, \
+     openquakeplatform_building_class_classificationhead AS h, \
+     openquakeplatform_building_class_classificationrow AS r, \
+     gadm_countries_simplified_1000m AS c \
+WHERE h.country = c.iso AND h.id = r.head_id AND h.owner_id = u.id AND h.owner_id NOT IN (15, 1820) ORDER BY u.username, h.id) \
+TO bcs_dump_${DT}.csv DELIMITER ',' CSV HEADER;" | tee -a $LOG_FILE | psql oqplatform
+
+
+attachments="../src/test/resources/my.pdf ../src/test/resources/my2.pdf"
+
+html_file="../src/main/resources/email_template.html"
+
+IFS="$old_ifs"
+
+# --------------------------------------------------------------------------------------
+# send html with pdf attachment
+# You can send additional attachments, the attachment list can be terminated with the "--"
+mutt -e "set copy=no" -e "set content_type=text/html" -s "$title" -a bcs_dump_${DT}.csv -- $GEM_BCS_REPORT < bcs_weekly-report_${DT}.html
+
+rm bcs_weekly-report_${DT}.html
+rm bcs_dump_${DT}.csv
```

Comment:
As above, I would use `/tmp` to avoid dust in Postgres' home in case of any failure.